messaging/src/main/java/org/axonframework/configuration/AbstractConfigurer.java

```diff
@@ -175,6 +176,14 @@ protected LifecycleSupportingConfiguration config() {
         return config;
     }
 
+    @Override
+    public void describeTo(@Nonnull ComponentDescriptor descriptor) {
+        descriptor.describeProperty("initialized", initialized.get());
+        components.describeTo(descriptor);
```

Comment:
Why not use `describeProperty` for this one? :thinking:

Reply:
Err, good question. I guess this is a brainfart on my end. I'll change it!

Reply:
I believe both will work :)